app/__tests__/stripe-webhook.test.ts

```diff
@@ -0,0 +1,440 @@
+import { describe, it, expect, vi, beforeEach } from "vitest";
```

Comment:
You need to pull and rebase again. When you do, prefer the patterns established on `main`